Packs/Troubleshoot/Scripts/ExecuteCommand/ExecuteCommand.yml

```diff
@@ -0,0 +1,78 @@
+args:
+- default: false
+  description: The command to run
+  isArray: false
+  name: command
+  required: true
+  secret: false
+- default: false
+  description: Argumnets to the command
+  isArray: false
+  name: arguments
+  required: false
+  secret: false
+- default: false
+  description: The instance name
+  isArray: false
+  name: instance_name
+  required: true
+  secret: false
+comment: Executes a command in CortexXSOAR with debug mode and pulls logs from the
+  command execution.
+commonfields:
+  id: ExecuteCommand
+  version: -1
+enabled: false
+name: ExecuteCommand
```

Comment:
for all scripts in this pack, lets add the suffix Troubleshoot

Reply:
done